src/kuzya.cpp

```diff
@@ -183,14 +140,68 @@ Kuzya::Kuzya(QWidget *parent)
 
     RFileList = new QList<QString>();
 
+    myFile = nullptr;
+
+    machine = new QStateMachine();
+    stateLanguageSelection = new QState(machine);
+    stateOfWritingCode = new QState(machine);
+
+    for(int i = 0; i < compiler->getSupportedLanguages().size(); i++)
```

Comment:
You could rewrite it with range-based for.

Reply:
Rewritten.